handsontable/src/shortcuts/manager.js

```diff
@@ -0,0 +1,107 @@
+import { createUniqueMap } from '../utils/dataStructures/uniqueMap';
+import { createContext } from './context';
+import { useRecorder } from './recorder';
+import { arrayEach } from '../helpers/array';
+
+export const createShortcutManager = ({ frame, beforeKeyDown, afterKeyDown }) => {
+  /**
+   * UniqueMap to storing contexts.
+   *
+   * @type {UniqueMap}
+   */
+  const CONTEXTS = createUniqueMap({
+    errorIdExists: keys => `The passed context name "${keys}" is already registered.`
+  });
+  /**
+   * Name of active context.
+   *
+   * @type {string}
+   */
+  let ACTIVE_CONTEXT_NAME = 'grid';
```

Comment:
I would suggest just do it in lower case as it isn't constant. WDYT? It's just some variable, used in scope. 

In the margin, I have been thinking about refactoring code to `class` instead of using current design pattern. It may be better even for docs. Maybe such considerations are good reason for doing it?

Reply:
As you said lower case, would be perfect for me here. About refactoring to `class` I wouldn't go there. I think that the factory function is clear enough.

Reply:
Changed within 516f4f5ba92c1a6b5b6d908f485e186f533f506c.